src/cls/IPM/General/SemanticVersionExpression.cls

```diff
@@ -57,14 +75,23 @@ Method ToResolvedString() As %String
     for i=1:1:..AnyOf.Count() {
       set opt = ..AnyOf.GetAt(i)
       set resolvedString = opt.ToResolvedString()
-      set optList = optList_$listbuild("(" _ resolvedString _ ")")
+      if ..AnyOf.Count() > 1 {
+        set optList = optList_$listbuild("(" _ resolvedString _ ")")
+      } else {
+        set optList = optList_$listbuild(resolvedString)
+      }
     }
     quit $listtostring(optList," || ")
   }
   if ..AllOf.Count() {
     for i=1:1:..AllOf.Count() {
       set opt = ..AllOf.GetAt(i)
-      set optList = optList_$listbuild("(" _ opt.ToResolvedString() _ ")")
+      set resolvedString = opt.ToResolvedString()
+      if ..AllOf.Count() > 1 {
+        set optList = optList_$listbuild("(" _ resolvedString _ ")")
```

Comment:
Why can parentheses not always be used? Seems simpler that way right? Both here and in ExpressionSet where you then won't need to strip the parentheses